lib/Service/FederationService.php

```diff
@@ -139,7 +139,7 @@ public function getRemoteRedirectURL(File $item, $direct = null) {
 				} else {
 					$wopi = $this->tokenManager->getRemoteTokenFromDirect($item, $direct->getUid());
 				}
-				$url = $remote . 'index.php/apps/officeonline/remote?shareToken=' . $item->getStorage()->getToken() .
+				$url = $remote . '/apps/officeonline/remote?shareToken=' . $item->getStorage()->getToken() .
```

Comment:
I would not cut out the `index.php`, as it should always work, whereas clean urls might not.

Compare with a similar line on richdocuments: https://github.com/nextcloud/richdocuments/blob/main/lib/Service/FederationService.php#L234